compiler/lib/src/main/scala/codegen/CppWriter/ComponentCppWriter/ComponentCppWriter.scala

```diff
@@ -905,6 +908,44 @@ case class ComponentCppWriter (
     }
   }
 
+  private def getDispatchCurrentMemebers: List[CppDoc.Class.Member] = {
+    val body = intersperseBlankLines(
```

Comment:
I think we don't need the `intersperseBlankLines` and we don't need the leading spaces in the body lines.